File: rb/utility/make_context.rb
# CrowdSourcedLyrics SDK utility: make_context
require_relative '../core/context'
module CrowdSourcedLyricsUtilities
  MakeContext = ->(ctxmap, basectx) {
    CrowdSourcedLyricsContext.new(ctxmap, basectx)
  }
end

File: php/utility/MakeContext.php
<?php
declare(strict_types=1);

// CrowdSourcedLyrics SDK utility: make_context

require_once __DIR__ . '/../core/Context.php';

class CrowdSourcedLyricsMakeContext
{
    public static function call(array $ctxmap, ?CrowdSourcedLyricsContext $basectx): CrowdSourcedLyricsContext
    {
        return new CrowdSourcedLyricsContext($ctxmap, $basectx);
    }
}
